server/controller/src/main/java/ai/starwhale/mlops/domain/runtime/RuntimeService.java

```diff
@@ -696,7 +696,7 @@ public BuildImageResult buildImage(String projectUrl, String runtimeUrl, String
             var user = userService.currentUserDetail();
             var image = new DockerImage(
                     dockerSetting.getRegistryForPull(),
-                    String.format("%s:%s", runtime.getName(), runtimeVersion.getVersionName())
+                    String.format("%s:%s", runtime.getName().toLowerCase(), runtimeVersion.getVersionName())
```

Comment:
How about adding some comments to let us know the reason for doing the `toLowerCase`

Reply:
done